spec/App/Controller/DashboardControllerSpec.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace spec\App\Controller;
+
+use App\Controller\DashboardController;
+use App\Dashboard\DashboardStatistics;
+use App\Dashboard\DashboardStatisticsProvider;
+use PhpSpec\ObjectBehavior;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Templating\EngineInterface;
+
+class DashboardControllerSpec extends ObjectBehavior
```

Comment:
There is no point in writing specs for controller. Let's test it with behat only